packages/shared/src/components/tooltips/Tooltip.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { ReactElement, useMemo } from 'react';
+import dynamicParent from '../../lib/dynamicParent';
+import { getShouldLoadTooltip, LazyTooltipProps } from './LazyTooltip';
+
+const LazyTooltipLoader = () =>
+  import(/* webpackChunkName: "lazyTooltip" */ './LazyTooltip');
+
+const TippyTooltip = dynamicParent<
+  Omit<LazyTooltipProps, 'render' | 'allowHTML'>
+>(() => LazyTooltipLoader().then((mod) => mod.default), React.Fragment);
+
+interface TooltipProps extends Omit<LazyTooltipProps, 'render'> {
+  disableTooltip?: boolean;
```

Comment:
What's this for?

Reply:
We have this specific component that accepts a parameter whether to enable or disable its tooltip, so I had to make a way to accommodate this scenario. But I can probably render conditionally instead of doing it in the tooltip itself to make more sense. 

https://github.com/dailydotdev/apps/blob/c9fe17977a660346d454a7bb5cf89c20b8315dcd/packages/shared/src/components/profile/ProfileLink.tsx